makefile

```diff
@@ -17,6 +17,17 @@ else
 	COVER_THRESHOLD=35.0 GOTEST_PKGS=./... ./lint-project.sh
 endif
 
+.PHONY: setup
+setup:
+	#mkdir -p specification
+	#wget -O specification/openapi.yaml https://raw.githubusercontent.com/moovfinancial/platform-api/master/go/platform/api/openapi.yaml?token={TOKEN}
```

Comment:
FYI the Moov API spec can be downloaded from a public source https://docs.moov.io/openapi/openapi.json

Reply:
thanks i was looking for that!